freeipa/src/main/java/com/sequenceiq/freeipa/util/KrbKeySetEncoder.java

```diff
@@ -0,0 +1,88 @@
+package com.sequenceiq.freeipa.util;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ActorKerberosKey;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+
+import org.bouncycastle.asn1.ASN1Encodable;
+import org.bouncycastle.asn1.DERInteger;
+import org.bouncycastle.asn1.DEROctetString;
+import org.bouncycastle.asn1.DERSequence;
+import org.bouncycastle.asn1.DERTaggedObject;
+
+public final class KrbKeySetEncoder {
+
+    private static final int TAG_0 = 0;
```

Comment:
if you need to have these to cheat checkstyle (ugh) then I think it would be better to at least name them something useful like TAG_MAJOR_VNO, TAG_MINOR_VNO, TAG_KVNO, etc corresponding to the field names.

Reply:
Thanks. This is good idea. Added this.